build/patches/Remove-signin-and-sync-integrations.patch

```diff
@@ -3719,16 +3719,48 @@ diff --git a/chrome/browser/password_manager/android/java/src/org/chromium/chrom
  import org.chromium.components.user_prefs.UserPrefs;
  import org.chromium.ui.base.WindowAndroid;
  
-@@ -81,17 +77,5 @@ public class PasswordManagerErrorMessageHelperBridge {
+@@ -49,29 +45,7 @@ public class PasswordManagerErrorMessageHelperBridge {
+      */
+     @CalledByNative
+     static boolean shouldShowErrorUi() {
+-        Profile profile = Profile.getLastUsedRegularProfile();
+-        final CoreAccountInfo primaryAccountInfo =
+-                IdentityServicesProvider.get().getIdentityManager(profile).getPrimaryAccountInfo(
+-                        ConsentLevel.SIGNIN);
+-        // It is possible that the account is removed from Chrome between the password manager
+-        // calling the Google Play Services backend and Chrome receiving the reply. In that
+-        // case, the error is no longer relevant/fixable.
+-        if (primaryAccountInfo == null) return false;
+-
+-        if (ChromeFeatureList.getFieldTrialParamByFeatureAsBoolean(
```

Comment:
I think it is irrelevant, this method should never be called

Reply:
sure, but the patch must also be useful, and it seems to me that it is not, today.

from what i'm understanding with the policy work, sync is disabled because the API key is missing, while signin is disabled if the user has not given consent and anyway it doesn't seem to me to be active without the MIRROR mode, that we have disabled.
the only case of calling google's servers, is related to the DICE mode which is only active in the non-android version, but I think this is more a bug than a will.
but I am still lacking certainty